source/module_xc/xc_functional.h

```diff
@@ -27,6 +27,8 @@ class XC_Functional
 	friend class LOOP_elec;
 	friend class Run_MD_PW;
 	friend class XCTest_PBE;
```

Comment:
Why the test class is placed in the friend class part? From my view, the main program may not containing codes related to testing.

Reply:
it's because the subroutines being tested are of static private type QwQ

Reply:
> it's because the subroutines being tested are of static private type QwQ

I suggest those subroutines being public members of the class.

Reply:
Sure, I've made the modifications